### PR TITLE
chore: update labels in 4 docs repos

### PR DESCRIPTION
**Pull request by yikeke**

### CODE REVIEW

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -455,8 +450,8 @@ ti-community-label:
       - 'special-week'
       - 'v1.0'
       - 'v1.1'
-      - 'v1.2'
-      - 'version-specific-changes-required'
+      - 'for-future-release'
```
Comment (ran-huang) [inline prow/config/external_plugins_config.yaml:453]: v1.2 should be retained. We'll have a v1.2-beta release next week.

**Comment by hi-rustin** [comment]
> @yikeke: The following tests **failed**, say `/retest` to rerun all failed tests:
> Test name 	Commit 	Details 	Rerun command
> pull-configs-validate-prow-yaml 	[643f9df](https://github.com/ti-community-infra/configs/commit/643f9df7b73c2c545257bfc27d24776012a7fcb0) 	[link](https://prow.tidb.io/view/gs/prow-tidb-logs/pr-logs/pull/ti-community-infra_configs/210/pull-configs-validate-prow-yaml/1374689616820441088) 	`/test pull-configs-validate-prow-yaml`
> pull-configs-validate-external-plugin-yaml 	[643f9df](https://github.com/ti-community-infra/configs/commit/643f9df7b73c2c545257bfc27d24776012a7fcb0) 	[link](https://prow.tidb.io/view/gs/prow-tidb-logs/pr-logs/pull/ti-community-infra_configs/210/pull-configs-validate-external-plugin-yaml/1374689616845606912) 	`/test pull-configs-validate-external-plugin-yaml`
> 
> [Full PR test history](https://prow.tidb.io/pr-history?org=ti-community-infra&repo=configs&pr=210). [Your PR dashboard](https://prow.tidb.io/pr?query=is%3Apr%20state%3Aopen%20author%3Ayikeke).

CI fails because there is a conflict in PR.

File: prow/config/external_plugins_config.yaml (hunk 1 above)
Comment (yikeke) [reply prow/config/external_plugins_config.yaml:453]:
```suggestion
      - 'v1.2'
      - 'for-future-release'
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**